docs/blog/2019-04-19-gatsby-why-we-write/index.md

```diff
@@ -0,0 +1,123 @@
+
+Each of these 70-80 messages & workflows probably has ~10 sub-workflows/sub-messages/features — imagine for each of these we had 3-5 case studies, docs pages, dedicated tutorials, starters/themes with annotated open-source source code -- that's thousands, probably tens of thousands, of pieces of content. _That's why we've only scratched the surface of what's possible._
+
+_(Now imagine a UI on top and the ability to mix & match!)_
+
+Our ideal version of Gatsby is a hyperconnected city — connected to all of these somewhat disparate communities (Wordpress, Drupal, modern frontend, accessibility), and so on.
+
+People from each of the communities who have discovered Gatsby are the experts on delivering the message of "why Gatsby" to that community.
+
+Our objective is **covering the ground** — or if you prefer, _breadth-first search_ rather than _depth-first search_ — making sure that each meaningful message has at least some basic material (both documentation of _how_ and the _why_ of case studies and success stories).
+
+In our content database, which lives in Airtable, we store an auto-updating column in the Messages table, which tracks the # of content pieces per message.
+
+But...there's a lot of ground to cover? **Yes, there is.**
+
+It's essential to our mission, though, because in order to become the default choice for building on the web, people need to realize that _Gatsby can do anything you conceivably want_ - this is why Wordpress became 30% of the internet! - and we emphasize that by writing about all use cases.
+
+_"Gatsby can't do X"_ gets rebutted by _"Here's a case study of Gatsby doing X"_.
+
+(For more background on this concept, read about ["economies of scope"](https://www.ribbonfarm.com/2012/10/15/economies-of-scale-economies-of-scope/) or think about companies that have flourished by providing access to literally everything their customers needed — the 1897 Sears Roebuck catalogue with [73K items](https://wpengine.com/reduce-reuse-refresh/); today, Amazon (the "Everything Store") with [100s of millions of items](https://wpengine.com/reduce-reuse-refresh/).)
+
+The challenge, is of course _how do we write all of that content?_
+
+### Principle 4: Leverage ourselves through collaboration.
+
+Some examples:
+
+- As an Inkteam, we're prioritizing podcasts to conference talks, because they take way less time and build on an existing community/listener base. The exception is key events like JAMStack and DrupalCon that are landmark events for specific communities.
```

Comment:
This minimizes the impact of some other big events we have going on this year, as well as speaking contributions of Inkteam members (Amberley, Dustin, Kyle, Preston, Sid, Ward, and anyone else who is presenting). For example, in 2019 there will be Frontend Masters workshops from two Gatsby team members (Jason and myself) and presentations at other high-profile conferences like An Event Apart. Is there a way to communicate priorities while also taking pride in the skills and growing industry reputations of our team members? How about something about making the most of each event to maximize awareness of Gatsby, with workshops, interviews, and Q&A?

Reply:
That's reasonable, and hints at a better way of phrasing this same concept -- because certainly _don't_ want to downplay that -- it's incredible work! Thanks for the suggestion.